Appveyor failing for Python 3.4 only

Not sure what's going wrong here. Looks like everything builds OK, but tests fail.

I wonder if we should throw a couple of easy checks in before the main testing. e.g. can numpy import? can scipy import? does file X exist?

https://ci.appveyor.com/project/snorfalorpagus/pywr/branch/master/job/358icypblkbtic1n

```
============================= test session starts =============================
platform win32 -- Python 3.4.5, pytest-3.0.5, py-1.4.31, pluggy-0.4.0 -- C:\Miniconda35-x64\conda-bld\pywr_1509456364200\_t_env\python.exe
cachedir: .cache
solver: glpk
rootdir: C:\Miniconda35-x64\conda-bld\pywr_1509456364200\test_tmp, inifile: 
collecting ... collected 4 items / 15 errors
=================================== ERRORS ====================================
_______________ ERROR collecting tests/test_agg_constraints.py ________________
ImportError while importing test module 'C:\Miniconda35-x64\conda-bld\pywr_1509456364200\test_tmp\tests\test_agg_constraints.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests\test_agg_constraints.py:1: in <module>
    from pywr.core import Model, Input, Output, Link, Storage, AggregatedNode, PiecewiseLink, MultiSplitLink
..\_t_env\lib\site-packages\pywr\core.py:5: in <module>
    from pywr.timestepper import Timestepper
..\_t_env\lib\site-packages\pywr\timestepper.py:3: in <module>
    from pywr import _core
pywr\recorders\_recorders.pxd:6: in init pywr._core (pywr/_core.c:46174)
    ???
..\_t_env\lib\site-packages\pywr\recorders\__init__.py:1: in <module>
    from .recorders import *
..\_t_env\lib\site-packages\pywr\recorders\recorders.py:5: in <module>
    from ._recorders import *
pywr\parameters\_parameters.pxd:5: in init pywr.recorders._recorders (pywr/recorders/_recorders.c:52233)
    ???
..\_t_env\lib\site-packages\pywr\parameters\__init__.py:1: in <module>
    from .parameters import *
..\_t_env\lib\site-packages\pywr\parameters\parameters.py:17: in <module>
    from scipy.interpolate import interp1d
..\_t_env\lib\site-packages\scipy\interpolate\__init__.py:176: in <module>
    from .interpolate import *
..\_t_env\lib\site-packages\scipy\interpolate\interpolate.py:21: in <module>
    import scipy.special as spec
..\_t_env\lib\site-packages\scipy\special\__init__.py:640: in <module>
    from ._ufuncs import *
E   ImportError: DLL load failed: The specified module could not be found.
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -15,11 +15,6 @@ environment:
       CONDA_INSTALL_LOCN: C:\\Miniconda-x64
 
     - TARGET_ARCH: x64
-      CONDA_PY: 34
-      CONDA_NPY: 111
-      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
-
-    - TARGET_ARCH: x64
       CONDA_PY: 36
       CONDA_NPY: 113
       CONDA_INSTALL_LOCN: C:\\Miniconda35-x64

